### ISSUE TITLE
Cyber engine tweaks Bug

### ISSUE THREAD

**Issue by gereen5** [maximegmd/CyberEngineTweaks#940]
cyber engine tweaks failed to resolve address for hash 0xf18b27a8. process will now close.

Since today, when I updated all the essential mods, when I start the game I get this error. Does the same happen to anyone else?

Help, please!

**Comment by AntonioKapuchino** [comment]
Same!

**Comment by maximegmd** [comment]
Can you try this version : https://github.com/maximegmd/CyberEngineTweaks/actions/runs/8374637248/artifacts/1346311910

**Comment by Kankro** [comment]
> Can you try this version : https://github.com/maximegmd/CyberEngineTweaks/actions/runs/8374637248/artifacts/1346311910

This version work 👍 , but the not v1.32.1 👎

**Comment by maximegmd** [comment]
This link is v1.32.1

**Comment by Kankro** [comment]
> This link is v1.32.1

Yes sorry i'm stupid, the error i get was for Red4Ext whit vortex when i try to install your package :) Sry.

**Comment by eisenbruch** [comment]
![image](https://github.com/maximegmd/CyberEngineTweaks/assets/6180294/4cfef91d-f025-4a8c-b31f-7a96ce3f4b74)
I get the above when I try to start cyberpunk 2077. just updated to CET 1.32.2
